execute/tokendata/usdc/http.go

```diff
@@ -108,14 +116,19 @@ func (r httpResponse) attestationToBytes() (cciptypes.Bytes, error) {
 
 func (h *httpClient) Get(ctx context.Context, messageHash cciptypes.Bytes32) (cciptypes.Bytes, HTTPStatus, error) {
 	// Terminate immediately when rate limited
-	if h.inCoolDownPeriod() {
+	if coolDown, duration := h.inCoolDownPeriod(); coolDown {
+		h.lggr.Errorw(
```

Comment:
I don't think this should be an error, maybe a warning?

Reply:
We have a self-rate limitting process in place which should always prevent from reaching coolDown. Cooldown is very destructive because the penalty of not respecting rate limits is high (e.g. backoff for 5 minutes).

Reply:
I don't have strong opinion on the logging level, but I'm okay with warn I guess

Reply:
No strong opinion either. Feel free to use what you think is better.